cvFit/src/app/macroCalculator/calculator.component.html

```diff
@@ -0,0 +1,310 @@
+                <mat-radio-button class="radio-button" *ngFor="let gen of genders" [value]="gen">{{gen}}</mat-radio-button>
+            </mat-radio-group>
+        </div>
+        <div class="form-group" id="hei">
+            <label for="height">Height:</label>
+            <input (ngModel)="heightF" name="heightF" required id="heightF" type="number" class="form-control" placeholder="5 foot" />
+            <input (ngModel)="heightI" name="heightI" required id="heightI" type="number" class="form-control" placeholder="10 inches" />
+        </div>
+        <div class="form-group">
+            <label for="weight">Weight:</label>
+            <input (ngModel)="weight" name="weight" required id="weight" type="number" class="form-control" placeholder="165" />
+        </div>
+        <div class="form-group">
+            <label for="activityLevel">Activity Level:</label>
+            <mat-form-field appearance="fill">
+                <mat-label>Activity Level</mat-label>
+                <mat-select [(ngModel)]="activity" name="activityLevel">
+                  <mat-option *ngFor="let activity of activityL" [value]="activity.value">
+                    {{activity.viewValue}}
+                  </mat-option>
+                </mat-select>
+              </mat-form-field>
+        </div>
+        <div class="form-group">
+            <label for="goal">Weight Goal:</label>
+            <mat-form-field appearance="fill">
+                <mat-label>Weight Goal</mat-label>
+                <mat-select [(ngModel)]="goal" name="weightGoal">
+                  <mat-option *ngFor="let goal of goalW" [value]="goal.value">
+                    {{goal.viewValue}}
+                  </mat-option>
+                </mat-select>
+              </mat-form-field>
+        </div>
+    
+        <div class="centerButton">
+            <button mat-raised-button color="primary" type="submit" [disabled]="macroForm.invalid">Calculate</button>
+            <button mat-stroked-button>Clear</button>
+        </div>
+        
+        
+    </form>
+</div> 
+<div class="info">
+    <h2>What are Macronutrients?</h2>
+    <p>In the context of health and fitness, macronutrients are most often defined to be the chemical compounds that humans consume in large quantities that provide bulk energy. 
+        Specifically, they refer to carbohydrates, proteins, and fats. Some definitions also include water, air, calcium, sodium, chloride ions, and some other substances, along 
+        with more typical macronutrients, since they are needed in large quantities by the human body. In this calculator, we only calculate daily carbohydrate, protein, and fat needs.</p>
+    <img src="./assets/images/info.jpg">
+
+    <h4><b>Review the chart below to find the macronutrient information you need to get started</b></h4>
+</div>  
+</div>    
+<div class='table-responsive'>
+    
+    <h2>Macronutrients in Common Foods</h2>
+    <table class='table'>
```

Comment:
You should think about if this belongs in its own component. If you think you might use this again or you want to break it apart for readability it might be a good idea.